models/hospital.js

```diff
@@ -0,0 +1,34 @@
+"use strict";
+const { Model } = require("sequelize");
+module.exports = (sequelize, DataTypes) => {
+  class Hospital extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+    }
+  }
+  Hospital.init(
+    {
+      hospital_uuid: {
+        type: DataTypes.UUID,
+        primaryKey: true,
+        allowNull: false,
+      },
+      hospitalname: { type: DataTypes.STRING.BINARY, allowNull: false },
+      recordcreatedsource: { type: DataTypes.STRING, allowNull: false },
+    },
+    {
+      sequelize,
+      timestamps: true,
+      createdAt: "recordcreatetimestamp", // set a custom name for createdAt timestamp
+      updatedAt: "recordupdatetimestamp", // set a custom name for updatedAt timestamp
+      tableName: "hospital",
+      modelName: "hospital",
```

Comment:
I think the convention is to have the modelName match the class name- in this case, capitalized Hospital.

Reply:
Good call